### ISSUE TITLE
Free Shipping filter option is visible even if none of my products are eligible

### ISSUE THREAD

**Issue by hoffmabc** [OpenBazaar/openbazaar-desktop#304]
Basically we should hide the Free Shipping option on the left sidebar if none of the items actually offer it. It's a tease.

![image](https://cloud.githubusercontent.com/assets/45482/21917507/33a7c898-d916-11e6-907f-9bab9ec661e5.png)

**Comment by rmisio** [comment]
@morebrownies Are you onboard with this?

**Comment by morebrownies** [comment]
I think we need to better define when it should be removed. 

If the store doesn't have a single listing that ships free to you, it probably makes sense to remove it all together since it will lead to a dead end anyways. 

When you perform a Search and none of the returned listings have Free Shipping, would you also expect for this option to hide @hoffmabc? Or if you drill down to a category that doesn't have any listings with free shipping, would you expect it hide?

@rmisio getting an estimate on level of effort would be helpful too once @hoffmabc provides more details on expected behavior across many scenarios.

**Comment by rmisio** [comment]
> If the store doesn't have a single listing that ships free to you, it probably makes sense to remove it all together since it will lead to a dead end anyways.

That's pretty easy to do. I'm not sure it's consistent with how we treat filters in general, e.g. if none of the listings ship to a specific country, we're still including all countries in the drop-down. But, if that's what we want, I'd be happy to do it.

> When you perform a Search and none of the returned listings have Free Shipping, would you also expect for this option to hide @hoffmabc? Or if you drill down to a category that doesn't have any listings with free shipping, would you expect it hide?

Dynamic filters are a rabbit hole we've intentionally avoided. If you did that for Free Shipping, then wouldn't you want to do it for everything else? For example, if I search for all listings that ship to Canada and those results are only represented by a few of the listed categories, do we update the categories filter? When other filters come, like price, do we update those?

I think we've taken the stance not to do that because it's a good amount of work to manage it and I'm not sure if the UX would be right with those filters changing like that.

**Comment by morebrownies** [comment]
>   I think we've taken the stance not to do that because it's a good amount of work to manage it and I'm not sure if the UX would be right with those filters changing like that.

I agree. It gets very complicated to manage and filters would be disappearing without warning. That could result in a more confusing experience for the user.

**Comment by morebrownies** [comment]
@hoffmabc are you ok with punting on this?

**Comment by hoffmabc** [comment]
Well Amazon greys it out I believe like if Prime is not available. Is that possible? I have not read the rest of this thread in detail if there's a long reason for why it can't be done.
